features/shared/checkout_previous_branch_after_success/deleted_current_branch/previous_branch_same.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Allow checking out the correct previous Git branch after running a Git Town commmand that keeps the previous branch intact and deletes the current branch
+
+  (see ../same_current_branch/previous_branch_same.feature)
+
+
+  Scenario: checkout previous branch after git-kill keeps the previous branch intact and deletes the current branch
+    Given I have feature branches named "previous" and "current"
+    And I am on the "previous" branch
+    And I checkout the "current" branch
+    And I run `git kill`
+    When I run `git checkout -` to checkout my previous Git branch
+    Then I end up on the "previous" branch
```

Comment:
I think this is more straightfoward with the additional step so we know where we are after the command ends
```cucumber
When I run `git kill`
Then I end up on the "main" branch
When I run `git checkout -` to checkout my previous Git branch
Then I end up on the "previous" branch
```